c_glib/arrow-glib/decimal128.cpp

```diff
@@ -456,8 +454,7 @@ garrow_decimal128_rescale(GArrowDecimal128 *decimal,
   auto arrow_decimal = garrow_decimal128_get_raw(decimal);
   arrow::Decimal128 arrow_rescaled_decimal_raw;
   auto status = arrow_decimal->Rescale(original_scale,
-                                       new_scale,
-                                       &arrow_rescaled_decimal_raw);
+                                       new_scale).Value(&arrow_rescaled_decimal_raw);
   if (garrow_error_check(error, status, "[decimal128][rescale]")) {
```

Comment:
Could you also use `garrow::check()` here?

```cpp
auto arrow_rescaled_decimal_raw = arrow_decimal->Rescale(original_scale, new_scale);
if (garrow_error_check(error, arrow_rescaled_decimal_raw, "[decimal128][rescale]")) {
  // ...
```

Reply:
Oh...